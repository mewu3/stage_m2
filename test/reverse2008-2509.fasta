> AY632535|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> HQ234498|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> KU720415|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> KU955594|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> LC002520|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> KU963573|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> KF383119|2008-2509
ugacguggggugcucaguggacuucucaaaaagagaaacgagauguggca
> KF383121|2008-2509
ugacguggggugcucaguggacuucucaaaaagagaaacgagauguggca
> KF383118|2008-2509
ugacguggggugcucaguggacuucucaaaaaaggaaacgagauguggca
> KF268948|2008-2509
ugacguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> KF268950|2008-2509
ugacguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> KF268949|2008-2509
ugacguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> KF383115|2008-2509
ugauguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> DQ859059|2008-2509
ugacguggggugcucgguggacuucucaaaaaaagaaacgagauguggca
> HQ234500|2008-2509
ugaugugggguguucgguggacuucucaaaaaaggaaacgagauguggca
> KU963574|2008-2509
ugaugugggguguucgguggacuucucaaaaaaggaaacgagauguggca
> HQ234501|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> KU955591|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> KU955592|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> KU955595|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> KX198134_(modified)|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> KF383116|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> KF383117|2008-2509
ugauguugggugcucgguggacuucucaaaaaaggaaacgagauguggca
> EU545988|2008-2509
ugauguggggugcucgguggacuucucaaagaaggaaacgagaugcggua
> JN860885|2008-2509
ugauguggggugcucgguggacuucucaaagaaggaaacgagaugcggua
> KU955593|2008-2509
ugauguggggugcucgguggacuucucaaagaaggaaacgagaugcggua
> KJ776791|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU991811|2008-2509
ugaugugggguguucgguggacuucucaaagaaggagacgagauguggua
> KU647676|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KU922923|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KU922960|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KU820897|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KX087102|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KX247646|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KX156774|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KX156775|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KX156776|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KX198135|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KU321639|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU509998|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX051563|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX197192|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX280026|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU527068|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU729218|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU312312|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU365778|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU758877|2008-2509
ugauguggggugcucgguggacuucucaaagaaggaaacgagaugcggua
> KU937936|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU501215|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU365777|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU365780|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU365779|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU707826|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX087101|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU501216|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU501217|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX262887|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX247632|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU870645|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU729217|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU926309|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU940224|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU940228|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU940227|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU926310|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU740184|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU761564|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU820898|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU955590|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX056898|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU820899|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX117076|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU866423|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX185891|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU963796|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU955589|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX253996|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU497555|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagauguggua
> KU853012|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU853013|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX101060|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KU744693|2008-2509
ugguguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KU681081|2008-2509
ugauguggggugcucgguggacuucucaaagaaggaaacgagaugcggua
> KX101061|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnngua
> KF993678|2008-2509
ugauguggggugcucgguggacuucucaaaaaaagaaacgagaugcggca
> KU681082|2008-2509
ugauguggggugcucgguggacuucucaaagaaggaaacgagaugcggua
> KX101065|2008-2509
nnnnnnggggugcucgguggacuucucaaagaaggagacgagaugmggua
> KX101064|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX101063|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> HQ234499|2008-2509
ugaugugggguguucgguggacuucucaaagaaggaaacgagaugcggua
> KX101066|2008-2509
ugauguggggugcucgguggacuucucaaagaaggagacgagaugcggua
> KX101062|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101067|2008-2509
nnnnnnnnnnnnnnnnnnnnacuucucaaagaaggagacgagaugcggua
> KF383120|2008-2509
agacguggggugcucaguugacuucgcuagaaaagagacaagauguggca
