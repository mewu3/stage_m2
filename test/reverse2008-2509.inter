> KF383119|2008-2509
ugacguggggugcucaguggacuucucaaaaagagaaacgagauguggca
> KF383121|2008-2509
ugacguggggugcucaguggacuucucaaaaagagaaacgagauguggca
> KF268948|2008-2509
ugacguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> KF268950|2008-2509
ugacguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> KF268949|2008-2509
ugacguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> KF383115|2008-2509
ugauguggggugcucgguggacuucucaaaaagagaaacgagauguggca
> DQ859059|2008-2509
ugacguggggugcucgguggacuucucaaaaaaagaaacgagauguggca
> KX101060|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101061|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnngua
> KF993678|2008-2509
ugauguggggugcucgguggacuucucaaaaaaagaaacgagaugcggca
> KX101063|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KX101062|2008-2509
nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn
> KF383120|2008-2509
agacguggggugcucaguugacuucgcuagaaaagagacaagauguggca
