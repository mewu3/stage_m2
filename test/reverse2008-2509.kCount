AAAAAAAGAAACG 2
AAAAAAGAAACGA 2
AAAAAGAAACGAG 2
AAAAAGAGAAACG 6
AAAAGAAACGAGA 2
AAAAGAGAAACGA 6
CAAAAAAAGAAAC 2
CAAAAAGAGAAAC 6
CAAAGAAGGAAAC 7
AAAGAAGGAAACG 7
AAGAAGGAAACGA 7
AGAAGGAAACGAG 7
TCTCGTTTCCTTC 7
AAAGAGAAACGAG 6
AAGAGAAACGAGA 6
