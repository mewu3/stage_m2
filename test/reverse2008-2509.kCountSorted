CAAAGAAGGAAAC 7
AAAGAAGGAAACG 7
AAGAAGGAAACGA 7
AGAAGGAAACGAG 7
TCTCGTTTCCTTC 7
AAAAAGAGAAACG 6
AAAAGAGAAACGA 6
CAAAAAGAGAAAC 6
AAAGAGAAACGAG 6
AAGAGAAACGAGA 6
AAAAAAAGAAACG 2
AAAAAAGAAACGA 2
AAAAAGAAACGAG 2
AAAAGAAACGAGA 2
CAAAAAAAGAAAC 2
