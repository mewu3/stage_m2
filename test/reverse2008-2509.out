AAGAAGGAGACGA	59
AAAAAAGGAAACG	16
CAAAGAAGGAAAC	7
